.npmrc

```diff
@@ -1,3 +1,3 @@
-@staffbase:registry=https://registry.npmjs.org/
+//registry.npmjs.org/:_authToken=${NODE_AUTH_TOKEN}
```

Comment:
This has effect on the local development. Better remove the file completely

Reply:
https://stackoverflow.com/questions/75766122/whats-the-difference-between-node-auth-token-and-npm-auth-token